presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/CanonicalizeExpressionRewriter.java

```diff
@@ -42,6 +45,38 @@ public static Expression canonicalizeExpression(Expression expression)
     private static class Visitor
             extends ExpressionRewriter<Void>
     {
+        @Override
+        public Expression rewriteComparisonExpression(ComparisonExpression node, Void context, ExpressionTreeRewriter<Void> treeRewriter)
```

Comment:
can you please add tests to `io.prestosql.sql.planner.iterative.rule.TestCanonicalizeExpressionRewriter`?